site/docs/v1/tech/apis/_new-api-key-401.adoc

```diff
@@ -0,0 +1,5 @@
+FusionAuth caches API keys for a faster lookup. There is an internal distributed cache and notification policy for multi-node deployments. When you create a new key, it will take a bit of time for the API key to be usable.
```

Comment:
Not sure we want to document specifics, but in practice if the multi-node communication is working it should be second or sub-second. 

Worse case scenario is that we can't talk to other nodes, and then we'd have a maximum of 60 seconds before the the key would be available on each node.

Reply:
I think it is fine to say "usually this notification occurs in less than one second. In rare cases where node communication fails, it may take up to 60 seconds." Will update.